Exiled.Events/EventArgs/ProcessingHotkeyEventArgs.cs

```diff
@@ -40,6 +43,11 @@ public ProcessingHotkeyEventArgs(Player player, HotkeyButton hotkey, bool isAllo
         /// </summary>
         public HotkeyButton Hotkey { get; }
 
+        /// <summary>
+        /// Gets or sets the item change to.
+        /// </summary>
+        public Item Item { get; set;  }
```

Comment:
The setter for this should check if the player actually possesses the item, and either throw an invalid operation error, or give them the item if they do not have it.